pull_blocks.py

```diff
@@ -56,5 +56,29 @@ def main():
         print(blockman)
         print("blocks pulled: %d" % blocks_pulled)
 
+
+class TestPullBlocks(unittest.TestCase):
+    def test_pull_blocks(self):
+        ctx = livectx
+        account = ctx["genesis_pub"]
+        peer = get_random_peer(ctx, lambda p: p.score == 1000)
```

Comment:
It should not be a random peer, we want to connect to a reliable peer.
Strictly speaking it should be a node controlled by us but let's use a well known peer for now.
Use 172.105.228.96